src/pretix/control/forms/admin.py

```diff
@@ -0,0 +1,45 @@
+from collections import OrderedDict
+
+from django.utils.translation import ugettext_lazy as _
+
+from pretix.base.forms import SettingsForm
+from pretix.base.i18n import I18nFormField, I18nTextInput
+from pretix.base.models.settings import AdminSetting
+from pretix.base.settings import SettingsProxy
+from pretix.base.signals import register_admin_settings
+
+
+class AdminDummyObject:
+    def __init__(self):
+        self.settings = SettingsProxy(self, type=AdminSetting)
+        self.slug = 'ADMINSETTINGS'
+
+    @property
+    def setting_objects(self):
+        return AdminSetting.objects
+
+
+class AdminSettingsForm(SettingsForm):
+    def __init__(self, *args, **kwargs):
+        self.obj = AdminDummyObject()
+        super().__init__(*args, obj=self.obj, **kwargs)
+
+        self.fields = OrderedDict([
+            ('footer_text', I18nFormField(
+                widget=I18nTextInput,
+                required=False,
+                label=_("Additional footer text"),
+                help_text=_("Will be included as additional text in the footer, site-wide.")
+            )),
+            ('footer_link', I18nFormField(
+                widget=I18nTextInput,
+                required=False,
+                label=_("Additional footer link"),
+                help_text=_("Will be included as the link in the additional footer text.")
+            ))
+        ])
+        responses = register_admin_settings.send(self)
+        for r, response in responses:
+            for key in response:
```

Comment:
I *think* it is this verbose because I tried using `.update()` only to discover that this will not preserve ordering, and then I jumped to the most verbose solution. Fixed.